includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -92,4 +98,47 @@ private function amp_content_add_auto_ads( $content ) {
 		);
 	}
 
+	/**
+	 * Web Story Ad Slot ID.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	private $story_ad_slot_id = '';
+
```

Comment:
Please, move this property declaration to be next to the existing properties of this class.